### PR TITLE
Gives the bicycle music 100% chance to play

### PR DESCRIPTION
**Pull request by improvedname**
Its stupid anyways, bicycles are rare in general and the music having a rare chance to play is stupid anyways considering it only plays clientside and not for others might aswell open youtube and play it

:cl: Improvedgay
tweak: Bike music now has 100% chance to play
/:cl:

### CODE REVIEW

**Comment by plapatin** [comment]
i want to ride my bicycle

File: code/modules/vehicles/bicycle.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "bicycle"
 	desc = "Keep away from electricity."
 	icon_state = "bicycle"
-	var/easter_egg_chance = 1
+	var/easter_egg_chance = 100
```
Comment (nicbn) [inline code/modules/vehicles/bicycle.dm]: If it won't be used, remove it and remove the checks.

**Comment by Iamgoofball** [comment]
:-1: this is an Easter Egg for a reason stop being an idiot, it plays every time you use it on April Fools already

**Comment by improvedname** [comment]
web edit boys

**Pull request closed without merging**